build/git-history.js

```diff
@@ -8,6 +8,12 @@ function getFromGit(contentRoot = CONTENT_ROOT) {
     cwd: contentRoot,
   });
 
+  // If `contentRoot` was a symlink, the `repoRoot` won't be. That'll make it
+  // impossible to compute the relative path for files within when we get
+  // output back from `git log ...`.
+  // So, always normalize to the real path.
+  const realContentRoot = fs.realpathSync(contentRoot);
```

Comment:
I would move this to the first line of this function and and use `realContentRoot` as `cwd` in the `execGit` to make it easier to understand.